app/helpers/home_helper.rb

```diff
@@ -33,15 +33,15 @@ def data_subheading(user, type)
   def data_path(user, type)
     if user.data_provider?
       case type
-      when "lettings" then lettings_logs_path(status: [:in_progress], assigned_to: "you")
-      when "sales" then sales_logs_path(status: [:in_progress], assigned_to: "you")
-      when "misc" then lettings_logs_path(status: [:completed], assigned_to: "you")
+      when "lettings" then lettings_logs_path(status: [:in_progress], assigned_to: "you", years: [""], owning_organisation_select: "all", managing_organisation_select: "all")
+      when "sales" then sales_logs_path(status: [:in_progress], assigned_to: "you", years: [""], owning_organisation_select: "all", managing_organisation_select: "all")
+      when "misc" then lettings_logs_path(status: [:completed], assigned_to: "you", years: [""], owning_organisation_select: "all", managing_organisation_select: "all")
       end
```

Comment:
Can we call clear filters in any way and then route them to the logs path with relevant filters only? Dw if now

Reply:
Not without refactoring the filter behaviour (as far as I can think of), which I didn't really want to do as part of this ticket, I think this is the cleanest solution.